yapapi/async_event_emitter.py

```diff
@@ -0,0 +1,28 @@
+import asyncio
```

Comment:
maybe just change the filename to `event_emitter.py` or `event_dispatcher.py` ?

Reply:
[b94e11c](https://github.com/golemfactory/yapapi/pull/866/commits/b94e11c2b0692c2b063834f588f142e068ac2f7a)